tests/integration_tests/test_llm.py

```diff
@@ -492,12 +493,46 @@ def test_default_max_sequence_length():
             BATCH_SIZE: 8,
             EPOCHS: 2,
         },
+        ADAPTER: {TYPE: "lora", PRETRAINED_WEIGHTS: "Infernaught/test_adapter_weights"},
+        BACKEND: {TYPE: "local"},
     }
     config_obj = ModelConfig.from_dict(config)
     assert config_obj.input_features[0].preprocessing.max_sequence_length is None
     assert config_obj.output_features[0].preprocessing.max_sequence_length is None
 
 
+def test_load_pretrained_adapter_weights():
```

Comment:
A couple of tests we should add (possibly in a followup PR):

- Checking a null input
- Checking an invalid weights path